### ISSUE TITLE
Release Process! (ROLLING LIVE ON JAMMY) :green_circle: :honey_pot: 

### ISSUE THREAD

**Issue by methylDragon** [ros-acceleration/ament_vitis#1]
## Buildfarm Status
- [![Build Status](https://build.ros2.org/job/Rdev__ament_vitis__ubuntu_jammy_amd64/badge/icon)](https://build.ros2.org/job/Rdev__ament_vitis__ubuntu_jammy_amd64/) **dev** (ubuntu_jammy_amd64)
- [![Build Status](https://build.ros2.org/job/Rdoc__ament_vitis__ubuntu_jammy_amd64/badge/icon)](https://build.ros2.org/job/Rdoc__ament_vitis__ubuntu_jammy_amd64/) **doc** (ubuntu_jammy_amd64)
- [![Build Status](https://build.ros2.org/job/Rsrc_el8__ament_vitis__rhel_8__source/badge/icon)](https://build.ros2.org/job/Rsrc_el8__ament_vitis__rhel_8__source/) **src** (rhel_8)
- [![Build Status](https://build.ros2.org/job/Rsrc_uJ__ament_vitis__ubuntu_jammy__source/badge/icon)](https://build.ros2.org/job/Rsrc_uJ__ament_vitis__ubuntu_jammy__source/) **src** (ubuntu_jammy)
- [![Build Status](https://build.ros2.org/job/Rbin_uJ64__ament_vitis__ubuntu_jammy_amd64__binary/badge/icon)](https://build.ros2.org/job/Rbin_uJ64__ament_vitis__ubuntu_jammy_amd64__binary/) **bin** (ubuntu_jammy_amd64)
- [![Build Status](https://build.ros2.org/job/Rbin_rhel_el864__ament_vitis__rhel_8_x86_64__binary/badge/icon)](https://build.ros2.org/job/Rbin_rhel_el864__ament_vitis__rhel_8_x86_64__binary/) **bin** (rhel_8_x86_64)
- [![Build Status](https://build.ros2.org/job/Rbin_ujv8_uJv8__ament_vitis__ubuntu_jammy_arm64__binary/badge/icon)](https://build.ros2.org/job/Rbin_ujv8_uJv8__ament_vitis__ubuntu_jammy_arm64__binary/) **bin** (ubuntu_jammy_arm64)

## Description
This issue tracks the status for the release of this repository into `rosdistro` and making it available for installation as `.deb` via the ROS 2 repositories.

It will detail the steps that need to be taken, and what recommended changes are good to have.

### Personal Note
I think this is ready for release, we just need to go through the release process. No changes required in the source that stand out to me.

### Steps
- [x] Create a **release tag** corresponding to the version specified in `package.xml` @vmayoral 
- [x] Create release repo (pending https://github.com/ros2-gbp/ros2-gbp-github-org/pull/22)
- [x] Wait for `rosdistro` merge of `ament_acceleration`
- [x] Create rolling and humble branches
- [x] Do [`bloom` release](https://wiki.ros.org/bloom/Tutorials/FirstTimeRelease)
  - [x] `bloom` should PR `rosdistro`, if not, **manually create a PR**
  - https://github.com/ros/rosdistro/pull/32595
- [x] `rosdistro` team to review package and merge PR (1-2 days, 1 month to get apt installable)
- [ ] Monitor build and release status on [repo page](http://repo.ros2.org/status_page/) and [buildfarm](https://build.ros2.org/)

### Recommendations
- It would be good to include some usage examples in the `README.md` for any downstream users of this package.

**Comment by vmayoral** [comment]
Reviewed the code accordingly, and added a few usage examples as suggested.

Pending on repo availability in release organization.

**Comment by methylDragon** [comment]
Good to go, pending release repo.

**Comment by methylDragon** [comment]
I can't generate the release files for this one until `ament_acceleration` gets merged into `rosdistro`, since it's specified as a dependency.

So this is pending the merge of https://github.com/ros/rosdistro/pull/32546

**Comment by methylDragon** [comment]
PR merged, we're ready to start the release process.

EDIT: I'll wait for `rolling` and `humble` branches for this repo, so we don't need to burden the ROS team with merging multiple PRs retargeting the branches, haha..

**Comment by vmayoral** [comment]
Branches available!

**Comment by methylDragon** [comment]
:tada:  https://github.com/ros/rosdistro/pull/32595

**Comment by methylDragon** [comment]
Merged!

**Comment by methylDragon** [comment]
@vmayoral I think the binary build is failing because https://github.com/ros-acceleration/ament_vitis/blob/main/package.xml is missing `ament_cmake_ros` as a dependency.

**Comment by methylDragon** [comment]
https://github.com/ros-acceleration/ament_vitis/commit/7fe7e6c768304577b08fe84bd442962021c31bd6 Nice, and I see the tag as well!

I'll try to update the release :crossed_fingers: hopefully it builds alright!

**Comment by methylDragon** [comment]
https://github.com/ros/rosdistro/pull/32649

**Comment by methylDragon** [comment]
Passed! :tada:

**Comment by vmayoral** [comment]
Thanks for the update!

**Comment by methylDragon** [comment]
Closing this issue, since we're done :tada: 
Congratulations on the press release too!
